lib/svgo.test.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+jest.mock('os', () => ({
+  EOL: '\r\n',
```

Comment:
I would prefer to avoid mocks and use only js2svg options

Reply:
The point of the mock is to test whether or not js2svg is defaulting to using the platform EOL; without a mock, we'd just be testing js2svg's behaviour when given options and not its platform-specific behaviour.

We could get rid of it if the test isn't all that important, or perhaps you have a better idea that I didn't think of?

Reply:
Better get rid of it. I will move os import to svgo-node.js so svgo.js import could be platform agnostic

Reply:
Addressed in latest push.